tomls/boba.rinkeby.toml

```diff
@@ -1,15 +1,26 @@
-[contracts]
+[pricefeeds]
 # ERC2362PriceFeed contracts
-pricefeeds = [
-  "0xa6ADEB16Cc5aB0d6A79957D9cf6234F508c33A74", # BtcUsdPriceFeed 
-  "0x542e8aD2e1AA508CC254E536DA98291776A15c65"  # EthUsdPriceFeed
+addresses = [
+  "0xeD074DA2A76FD2Ca90C1508930b4FB4420e413B0", # BtcUsdPriceFeed 
+  "0xD9465D38f50f364b3263Cb219e58d4dB2D584530", # EthUsdPriceFeed
+  "0x56834Ff8D4b27db647Da97CA3bd8540f7fA0e89D", # OmgBtcPriceFeed
+  "0x225BAd150B9D5202DC805B34A0DF64B1a77459dF", # OmgEthPriceFeed
+  "0xE2Efa3fe66352e63F118bB9165435C5BEDB777d0"  # OmgUsdtPriceFeed
+]
+thresholds = [
+  2.0, # BtcUsdPriceFeed
+  2.0, # EthUsdPriceFeed
+  2.0, # OmgBtcPriceFeed
+  2.0, # OmgEthPriceFeed
+  2.0  # OmgUsdtPriceFeed
```

Comment:
Can we do something like this instead?
```toml
[pricefeeds]
0xeD074DA2A76FD2Ca90C1508930b4FB4420e413B0 = 2.0
```

Or like this? (also possible as per TOML specs)
```toml
[pricefeeds]
[pricefeeds-BtcUsdPriceFeed]
address = "0xeD074DA2A76FD2Ca90C1508930b4FB4420e413B0"
threshold = 2.0
```

Otherwise this is a disaster waiting to happen!